tools/validators/instance_validator/validate/handler.py

```diff
@@ -29,32 +29,31 @@
 DEFAULT_TIMEOUT = 600
 
 
-def deserialize(yaml_files, universe):
```

Comment:
Looks like the change that made it possible to validate multiple files at once is being undone here. Please merge / resolve conflicts so we don't lose that work.